TestFramework/ExtensionMethods.cs

```diff
@@ -153,11 +152,10 @@ public static bool IsElementVisible(IWebDriver driver, By by, ScenarioContext sc
                 driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(0);
                 WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(0));
                 wait.Until(ExpectedConditions.ElementIsVisible(by));
-                return driver.FindElement(by).Displayed && driver.FindElement(by).Enabled;
```

Comment:
Why remove this?  I recall that there was an issue with displayed but not yet enabled.

Reply:
There was a method added for isElementEnabled which checks for Displayed and Enabled. I removed this as it was failing in some cases as this method (isElementVisible) was initially only intended to check for 'Displayed'